CppCoreGuidelines.md

```diff
@@ -21018,7 +21018,7 @@ However, we have had many requests for a set of naming and layout conventions to
 
 More specific and detailed rules are easier to enforce.
 
-These rules bear a strong resemblance to the recommendations in the [PPP Style Guide](http://www.stroustrup.com/Programming/PPP-style.pdf)
```

Comment:
We're not sure what problem there is with the original (identical twins bear a strong resemblance to each other), and we think the original is clearer.

Reply:
reverted